tfjs-core/src/index.ts

```diff
@@ -101,18 +108,35 @@ export {
   test_util,
   util,
   backend_util,
+  complex_util,
   webgl,
   tensor_util,
   slice_util,
   gather_util,
-  scatter_util
+  scatter_util,
```

Comment:
Curious why you've changed it to export these explicitly rather than through backend_util?